registry/client/transport/http_reader.go

```diff
@@ -163,10 +155,8 @@ func (hrs *httpReadSeeker) reader() (io.Reader, error) {
 	}
 
 	if hrs.readerOffset > 0 {
-		// TODO(stevvooe): Get this working correctly.
-
 		// If we are at different offset, issue a range request from there.
-		req.Header.Add("Range", "1-")
+		req.Header.Add("Range", "bytes="+strconv.FormatInt(hrs.readerOffset, 10)+"-")
```

Comment:
Is there a gain to using this over `fmt.Sprintf`? `fmt.Sprintf` is certainly easier to read